src/App.tsx

```diff
@@ -116,6 +116,18 @@ function App(): ReactElement {
 
   const [isRecording, setIsRecording] = useState(false);
   const timeControls = useConstructor(() => new TimeControls(canv!, playbackFps));
+  // TODO: Move all logic for the time slider into its own component.
```

Comment:
I was tempted to do this as part of this PR, but I think the changes are much more readable if I don't touch App just yet.